core/src/main/java/io/opentelemetry/android/features/diskbuffering/DiskBufferingConfig.kt

```diff
@@ -15,6 +15,7 @@ const val MAX_CACHE_FILE_SIZE: Int = 1024 * 1024
 const val DEFAULT_MAX_FILE_AGE_FOR_WRITE_MS = 30L
 const val DEFAULT_MIN_FILE_AGE_FOR_READ_MS = 33L
 const val DEFAULT_MAX_FILE_AGE_FOR_READ_MS = 18L
+const val DEFAULT_EXPORT_SCHEDULE_DELAY_MILLIS: Long = 10000L // 10 seconds
```

Comment:
```suggestion
const val DEFAULT_EXPORT_SCHEDULE_DELAY_MS: Long = 10000L
```
Following same convention